### ISSUE TITLE
build failure under macOS

### ISSUE THREAD

**Issue by xen0n** [xen0n/autojump-rs#1]
```
error[E0433]: failed to resolve. Use of undeclared type or module `autojump`
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/autojump-0.2.1/src/data/datafile.rs:18:23
   |
18 |     let xdg_aj_home = autojump::xdg_home_hardcoded();
   |                       ^^^^^^^^ Use of undeclared type or module `autojump`

error[E0308]: mismatched types
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/autojump-0.2.1/src/data/datafile.rs:20:9
   |
20 |         Ok(())
   |         ^^^^^^- help: try adding a semicolon: `;`
   |         |
   |         expected (), found enum `std::result::Result`
   |
   = note: expected type `()`
              found type `std::result::Result<(), _>`

error[E0308]: mismatched types
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/autojump-0.2.1/src/data/datafile.rs:17:60
   |
17 |   fn migrate_osx_xdg_data(config: &Config) -> io::Result<()> {
   |  ____________________________________________________________^
18 | |     let xdg_aj_home = autojump::xdg_home_hardcoded();
19 | |     if !xdg_aj_home.exists() {
20 | |         Ok(())
...  |
29 | |     fs::remove_file(old_config.backup_path)?;
30 | | }
   | |_^ expected enum `std::result::Result`, found ()
   |
   = note: expected type `std::result::Result<(), std::io::Error>`
              found type `()`

error: aborting due to 3 previous errors
```

**Comment by tomv564** [comment]
Hi, great initiative! Are you interested in a PR for this?

**Comment by xen0n** [comment]
Of course, please go ahead!

It shouldn't be too hard I think... I never got any time to implement this myself (sorry for that), partly because I don't use `j` very often at work (almost keeping one tmux pane per directory). :man_facepalming:

**Comment by xen0n** [comment]
Closed by #3.
